docs/getting-started/installation.md

```diff
@@ -145,6 +154,14 @@ When you run the command above, a directory called .venv will appear in the fold
 
 This initializes a template app in your new directory.
 
+
+5. Run the demo app to get an idea of what you can build with Reflex:
```

Comment:
Should we consolidate these last portions since they're not specific to the environment?